### PR TITLE
Correct numerous typos and inconsistencies in manual.

### PR DESCRIPTION
**Pull request by pkirkham**
Correcting typos. Spelling, consistent usage of parameter identifiers, correct description of units (confirmed using ECLIPSE manual).

### CODE REVIEW

**Comment by gdfldm** [comment]
@pkirkham, thank you for all the corrections. That all looks great. The only thing I noticed was an extra E that has been added on to the end of GCONINJE in WCONINJE.fodt just below the table.

File: parts/chapters/subsections/12.3/WCONINJE.fodt [hunk 1]
```diff
@@ -5235,7 +5235,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
     </table:table>
     <text:p text:style-name="P70">Table <text:sequence text:ref-name="refTable0" text:name="Table" text:formula="ooow:Table+1" style:num-format="1">12.78</text:sequence>: WCONINJE Keyword Description</text:p>
     <text:p text:style-name="P74"/>
-    <text:p text:style-name="P72">See also the GCONPROD the GCONINJ<text:span text:style-name="T54">E</text:span> keywords to define a group’s production and injection targets and constraints, and the WCONPROD keyword to define a production well’s targets and constraints. All the aforementioned keywords are described in the SCHEDULE section.</text:p>
+    <text:p text:style-name="P72">See also the GCONPROD the GCONINJE<text:span text:style-name="T54">E</text:span> keywords to define a group’s production and injection targets and constraints, and the WCONPROD keyword to define a production well’s targets and constraints. All the aforementioned keywords are described in the SCHEDULE section.</text:p>
```
Comment (gdfldm) [inline parts/chapters/subsections/12.3/WCONINJE.fodt:5238]: Please delete extra E

**Review by gdfldm — APPROVED** [review]
Just one nit found. Ok to merge and fix later.

File: parts/chapters/subsections/12.3/WCONINJE.fodt (hunk 1 above)
Comment (pkirkham) [reply parts/chapters/subsections/12.3/WCONINJE.fodt:5238]: Oops. Will change that. All these spans in the LibreOffice document can make it really hard to see what is going on properly sometimes.

Comment (blattms) [reply parts/chapters/subsections/12.3/WCONINJE.fodt]: Thanks. I totally agree about spans.

Comment (hakonhagland) [reply parts/chapters/subsections/12.3/WCONINJE.fodt]: > I totally agree about spans.

@blattms Maybe we can write a script to remove all those span tags? I think they are only used for version control https://help.libreoffice.org/latest/en-US/text/shared/guide/redlining_versions.html, but we are already using Git for version control so we should not need them, right?

**Comment by blattms** [comment]
thank you so much.

**Pull request merged**